src/pageContent/Profile/index.tsx

```diff
@@ -0,0 +1,104 @@
+import Text from "src/components/Text";
+import Button from "src/components/Button";
+import Input from "src/components/Input";
+import { useState } from "react";
+
+const ProfileContent: React.FC<Record<string, never>> = () => {
+  const [name, setName] = useState<string>(undefined);
+  const [subject, setSubject] = useState<string>(undefined);
+  const [content, setContent] = useState<string>(undefined);
+
+  const handleSubmit = (event: React.MouseEvent) => {
+    event.preventDefault();
+
+    if (!name) {
+      setName(""); // Lazy way to trigger errror
+      return;
+    }
+    if (!subject) {
+      setSubject(""); // Lazy way to trigger errror
+      return;
+    }
+    if (!content) {
+      setContent(""); // Lazy way to trigger errror
+      return;
+    }
+
+    const body =
+      `Sup Amadeus,\n\n` +
+      `${content}\n\n` +
+      `Thank you.\n\n` +
+      `Best regards,\n\n` +
+      `${name}\n`;
+
+    // Not too sure why the below needs encodeURIComponent. Think because its a URL? 
+    // Oh it doesn't actually send the email just to open the email App, if any. 
+    // What happens if don't have (?)
+    location.href =
```

Comment:
hem maybe you can try to explain what does `locatin.href` means, what does` mailto:` mean as an url

Reply:
Hem mailto: stau gw itu link yg kalo lu click dia ngebuka email app ? Location.href supposedly nyimpen current url I think y? Cmn yg ini gw g bgitu ykain kyk kalo g ad app yg bs nge-email ini bkl ngapain. Sama sequence nya gmn kyk kalo lu ngewrite k location.href trs dia blocking kah smp email app buka ato gmn gmn. Iy basically gw g bgitu yakin location.href setup nya gmn lel ntar gw update lgi

Reply:
mailto itu URI, yah bisa dibilang sejenis link (engga sama sih sbnrny), sama iya location.href itu nyimpen metadata of current uri dari browsernya. engga, kalo g ad urinya, dia throw error doank, itu sama aja kayak ngebuka invalid uri